template/README.md

```diff
@@ -404,6 +405,44 @@ Note that GitHub Pages doesn't support routers that use the HTML5 `pushState` hi
 
 Use the [Heroku Buildpack for create-react-app](https://github.com/mars/create-react-app-buildpack).
 
+### Adding Custom Environment Variables
+
+> Note: this feature is available with `v0.3.0` and higher.
+
+Your project can consume variables declared in your environment as if they were declared locally in your JS files. By
+default you will have `NODE_ENV` defined for you, and any other environment variables starting with
+`REACT_APP_`. These environment variables will be defined for you on `process.env`. For example, having
+`REACT_APP_SECRET_CODE=abcdef` will be exposed in your JS as `process.env.REACT_APP_SECRET_CODE`, along with
+`process.env.NODE_ENV`.
+
+These environment variables can be useful for displaying information conditionally based on where the project is
+deployed or consuming sensitive data that lives outside of version control.
+
+```bash
+$ REACT_APP_SECRET_CODE=abcdef
+$ npm start
+```
+
+```jsx
+<h1>Hello, Admin!<h1>
+
+<small>You are current running this application in <strong>{process.env.NODE_ENV}</strong> mode.</small>
+
+<form>
+  <input type="hidden" defaultValue={process.env.REACT_APP_SECRET_CODE} />
+</form>
+```
+
+Having access to the `NODE_ENV` is also useful for loading different JS files or views based on the value:
+
+```jsx
```

Comment:
I think we’d like to avoid showing this pattern for now because we try to use ES6 imports where possible, and people might get confused.

I would rather like to show a regular `if` condition, like this:

```
if (process.env.NODE_ENV !== 'production') {
  analytics.disable();
}
```